requirements.txt

```diff
@@ -1,50 +1,73 @@
 asgiref==3.7.2
+attrs==23.2.0
+autobahn==23.6.2
+Automat==22.10.0
+awsebcli==3.20.10
+bcrypt==4.1.2
 black==24.2.0
 blessed==1.20.0
 boto==2.49.0
 boto3==1.34.73
-botocore==1.34.73
+botocore
 cement==2.8.2
 certifi==2024.2.2
+cffi==1.16.0
+channels==4.1.0
+chardet==5.2.0
 charset-normalizer==3.3.2
 click==8.1.7
 colorama==0.4.3
-cryptography==42.0.5
+constantly==23.10.4
 coverage==6.5.0
 coveralls==3.3.1
+cryptography==42.0.5
+daphne==4.1.1
 distlib==0.3.8
 Django==4.2
 django-storages==1.14.2
 docopt==0.6.2
 filelock==3.13.1
 flake8==7.0.0
+hyperlink==21.0.0
 idna==3.6
+incremental==22.10.0
 jmespath==1.0.1
 mccabe==0.7.0
 mypy-extensions==1.0.0
+mysql-connector==2.2.9
+mysql-connector-python==8.3.0
```

Comment:
Please remove these two packages
mysql-connector==2.2.9
mysql-connector-python==8.3.0

They cause issues with the deployment